### PR TITLE
Update master with beta.

### PR DESCRIPTION
**Pull request by richardfazzio**
Update README.md and unit tests.
**NOTE** Unit tests will fail on master due to the changes for addLabels clientapihandler that was fixed in beta's circuit.js. Also, the formSubmission is broken on the backend.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,15 @@
 Circuit JavaScript and Node.js SDK
 ==================================
 
-[![GitHub release](https://img.shields.io/github/release/circuit/circuit-sdk.svg)](https://github.com/circuit/circuit-sdk)
-[![Build Status](https://travis-ci.org/circuit/circuit-sdk.svg?branch=master)](https://travis-ci.org/circuit/circuit-sdk)
+[![NPM release](https://img.shields.io/npm/v/circuit-sdk/beta.svg)](https://www.npmjs.com/package/circuit-sdk)
```
Comment (rogeru) [inline README.md]: For master you need to keep the release badges, not the beta ones.

File: README.md [hunk 2]
```diff
@@ -1,13 +1,15 @@
...
+[![Build Status](https://travis-ci.org/circuit/circuit-sdk.svg?branch=beta)](https://travis-ci.org/circuit/circuit-sdk)
```
Comment (rogeru) [inline README.md]: Same

File: README.md [hunk 3]
```diff
@@ -1,13 +1,15 @@
...
 [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
 
 
 ### Prerequisites
-* Register for a free developer account and create your own OAuth app to get a `client_id`. See [Getting Started](https://circuit.github.io/) for details.
+* Developer account on circuitsandbox.net. Get it for free at [developer registration](https://www.circuit.com/web/developers/registration).
```
Comment (rogeru) [inline README.md]: Keep what we had in master. I think that was updated there

File: README.md [hunk 4]
```diff
@@ -1,13 +1,15 @@
...
+* OAuth 2.0 `client_id` and optionally `client_secret`. Get if for free at [circuit.github.com/oauth](https://circuit.github.com/oauth).
```
Comment (rogeru) [inline README.md]: Same

File: README.md [hunk 5]
```diff
@@ -17,14 +19,14 @@ https://circuitsandbox.net/sdk/ with most APIs described in the [Client](https:/
 #### JavaScript
 Add the the line below to the HTML file of your app. This will include the latest version of Circuit SDK and make the `Circuit` object available.
 ```html
-<script type="text/javascript" src="https://unpkg.com/circuit-sdk"></script>
+<script type="text/javascript" src="https://unpkg.com/circuit-sdk@beta"></script>
```
Comment (rogeru) [inline README.md]: Don't use beta

File: README.md [hunk 6]
```diff
@@ -17,14 +19,14 @@ https://circuitsandbox.net/sdk/ with most APIs described in the [Client](https:/
...
 ```
 
-> The JS SDK can now also be imported as CommonJS or AMD module. See [umd](https://github.com/circuit/umd) or [circuit-ionic-starter](https://github.com/circuit/circuit-ionic-starter) repos for examples.
+JS SDK can also be imported via CommonJS or AMD. View the repo [UMD](https://github.com/circuit/umd) for the different types of usage, or [circuit-ionic-start](https://github.com/circuit/circuit-ionic-starter) for example how import the SDK via `require` using webpack.
 
 #### Node.js
 ```bash
-npm install --save circuit-sdk
+npm install --save circuit-sdk@beta
```
Comment (rogeru) [inline README.md]: no beta on master

File: README.md [hunk 7]
```diff
@@ -33,7 +35,7 @@ The majority of apps fall in one of the three types below. Each type uses a diff
 1. **Client-side web applications** use the JavaScript SDK with the [OAuth 2.0 Implicit Grant](https://circuit.github.io/oauth.html#implicit).
    ```javascript
    // No client_secret needed for Implicit Grant. SDK will obtain access token.
-   const client = new Circuit.Client({
+   let client = new Circuit.Client({
```
Comment (rogeru) [inline README.md]: Any reason we change it to let?

File: test/browser/whiteboard.js [hunk 8]
```diff
@@ -171,16 +171,6 @@ describe('Whiteboard tests', async function() {
         assert(!whiteboard.elements);
     });
 
-    it('function: undoWhiteboard, raises event: whiteboardSync', async () => {
```
Comment (rogeru) [inline test/browser/whiteboard.js]: Are you removing this?

File: README.md (hunk 7 above)
Comment (richardfazzio) [reply README.md]: I'm not sure, const works I will change it back. Should I change it to const on beta as well?

File: test/browser/whiteboard.js (hunk 8 above)
Comment (richardfazzio) [reply test/browser/whiteboard.js]: Yes, it was removed on beta because it was a wrong test.

**Comment by richardfazzio** [comment]
Fork was deleted, creating a new repo

**Pull request closed without merging**